src/components/FieldValueDirectAnswer.tsx

```diff
@@ -58,7 +65,19 @@ export function FieldValueDirectAnswer({
   );
 }
 
-function getResultContent(result: FieldValueDirectAnswerType): JSX.Element {
+function DefaultUnknownFieldTypeDisplay({ result }: { result: UnknownFieldValueDirectAnswer }): JSX.Element {
+  if (typeof result.value !== 'string' && typeof result.value !== 'number') {
+    console.warn(`Unknown field type for direct answer with "${result.fieldApiName}" fieldApiName. Rendering result's value as a string.`
+      + '\nConsider using prop "UnknownFieldTypeDisplay" in DirectAnswer component to properly render result of unknown field type.');
+  }
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return getTextJsxElement((result.value as any).toString());
```

Comment:
Would we ever wanna use JSON.stringify here?

Reply:
sure, that would provide a bit more info to user, even if it looks a bit ugly, which would prompt developer to provide UnknownFieldTypeDisplay.